### ISSUE TITLE
all countries not available for trending

### ISSUE THREAD

**Issue by pramod-lamsal** [jonjomckay/fritter#137]
First of all thank you all for making such a beautiful app .It will be better for the users living beside these few countries if you guys will add trending support of more countries.

**Comment by jonjomckay** [comment]
Unfortunately, I'm only able to display the trend locations that Twitter gives me back through the API, and it seems like it's quite a limited list of countries...

Is it possible to view trends for a country that isn't in the list Fritter displays? In the official app, maybe? If so, I can try and figure out how Fritter can support them too.

**Comment by pramod-lamsal** [comment]
Yes it is available in the official app.Hope this feature will be soon available in fritter also so that we can completely dump Twitter official app.

**Comment by NabilAKROUD** [comment]
Same for Morocco 🇲🇦
